README.adoc

```diff
@@ -289,15 +302,29 @@ InventoryConfig.java
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryConfig.java[]
 ----
+
 Inject the [hotspot=custom-converter file=3]`io_openliberty_guides_email` property, and add the [hotspot=getEmail file=3]`getEmail()` method.
 
+Repeat the same in `SystemConfig`.
+[role="code_command hotspot file=4", subs="quotes"]
+----
+#Replace the `SystemConfig` class.#
+`src/main/java/io/openliberty/guides/system/SystemConfig.java`
+----
+SystemConfig.java
+[source, Java, linenums, role='code_column hide_tags=copyright']
+----
+include::finish/src/main/java/io/openliberty/guides/system/SystemConfig.java[]
+----
+Inject the [hotspot=custom-converter file=4]`io_openliberty_guides_email` property, and add the [hotspot=getEmail file=4]`getEmail()` method.
```

Comment:
On second thought, I think if you say "Inject the `io_openliberty_guides_email` property, and add the `getEmail()` method." you can then say in the next sentence "Inject the same properties into the `SystemConfig.java` class". Then you don't need the second "Inject the `io_openliberty_guides_email` property, and add the `getEmail()` method" as it is redundant.

Reply:
I made some changes what do you think now?